src/index.html

```diff
@@ -10,23 +10,49 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Mate academy | airbnb search bar</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
-  </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
+    <link
+      rel="stylesheet"
+      href="search-bar.css"
     />
+  </head>
+  <body class="page">
+    <form
+      class="form page__form form--big"
+      action="GET"
+      data-qa="big"
+    >
+      <label
+        class="label label--big"
```

Comment:
You have created so many blocks like label, form, page. This task can be done with using only one block.